concent_api/core/tests/test_auth_accept_or_reject_results.py

```diff
@@ -20,11 +20,22 @@
     CONCENT_PUBLIC_KEY        = CONCENT_PUBLIC_KEY,
     CONCENT_MESSAGING_TIME    = 10,  # seconds
     FORCE_ACCEPTANCE_TIME     = 10,  # seconds
-    SUBTASK_VERIFICATION_TIME = 10,  # seconds
     CONCENT_ETHEREUM_ADDRESS  = 'x' * ETHEREUM_ADDRESS_LENGTH
 )
 class AuthAcceptOrRejectIntegrationTest(ConcentIntegrationTestCase):
 
+    def setUp(self):
+        super().setUp()
+        self.patcher_1 = mock.patch('core.message_handlers.calculate_subtask_verification_time', return_value=10)
```

Comment:
Actually there is one pitfall patching this way. If for some reason a test encounters an error (not fails, but breaks/crashes) `tearDown` method will not be called for it, thus patch will remian 'unstopped'. To overcame this, you could rewrite the setup to sth like:
```
def setUp(self):
        super().setUp()
        patcher_in_message_handlers = mock.patch('core.message_handlers.calculate_subtask_verification_time', return_value=1800)
        patcher_in_transfer_operations = mock.patch('core.transfer_operations.calculate_subtask_verification_time', return_value=1800)
        patcher_in_message_handlers.start()
        patcher_in_transfer_operations.start()
        self.addCleanup(patcher_in_message_handlers.stop)
        self.addCleanup(patcher_in_transfer_operations.stop)
```
and then you do not have to overwrite `tearDown` and you don't need `self.moc_foo`.
Reference: https://docs.python.org/3/library/unittest.html#unittest.TestCase.addCleanup
Please apply this approach to another test classes as well

Reply:
This might be a good addition to our style guide. It's meant to list common pitfalls like this.

Reply:
> Cleanup items are called even if setUp fails (unlike tearDown)
my mistake, I will edit first comment

Reply:
:+1: done